recipes/k3d/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "k3d" %}
+{% set version = "2.6.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name|upper }}-{{ version }}.tar.gz
+  sha256: 994438e8f73e82d683db79092ab5c8698b8bd4b1e077914853b6d14800aacb22
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - ipywidgets >=7.0.1
+    - ipydatawidgets
+    - traittypes
+    - traitlets
+    - numpy >=1.11.0
+test:
+  commands:
+    - cd {{ SRC_DIR }}/js && npm install && xvfb-run -a grunt test  # [linux]
+  requires:
+    - grunt
+    - nodejs
+
+about:
+  home: http://github.com/K3D-tools/K3D-jupyter
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
+  summary: 'Jupyter notebook extension for 3D visualization.'
+  dev_url: https://github.com/K3D-tools/K3D-jupyter
+
+extra:
+  recipe-maintainers:
+    - renefritze
+    - tgandor
+    - artur-trzesiok
+    - marcinofulus
```

Comment:
@marcinofulus @artur-trzesiok @tgandor - are you OK with being maintainers?

Reply:
I think we already covered this following https://github.com/conda-forge/staged-recipes/pull/8050#issuecomment-477044107 ?

Reply:
yep!